src/components/EntityStatusNew/EntityStatus.tsx

```diff
@@ -21,6 +21,7 @@ const EFlagToLabelTheme: Record<EFlag, LabelProps['theme'] | 'orange'> = {
     [EFlag.Grey]: 'unknown',
     [EFlag.Orange]: 'orange',
     [EFlag.Yellow]: 'warning',
+    [EFlag.DarkGrey]: 'unknown',
```

Comment:
but we never get such status from backend - it's not presented in proto https://github.com/ydb-platform/ydb/blob/b97b42caffd37e4a5462751d2336b9df9adc84ab/ydb/core/viewer/protos/viewer.proto#L312